client/app/dedicatedUniverseComponents/notification/index.js

```diff
@@ -0,0 +1,15 @@
+import angular from 'angular';
+
+import 'ovh-angular-user-pref';
+
+import ducNotification from './notification.service';
+
+const moduleName = 'ducNotification';
+
+angular
+  .module(moduleName, [
+    'ovh-angular-user-pref',
+  ])
+  .service('DucNotification', ducNotification);
```

Comment:
Please refer to my previous comment:

```suggestion
  .service('DucNotification', DucNotification);
```